analyzer/src/main/kotlin/managers/Unmanaged.kt

```diff
@@ -34,7 +34,9 @@ import org.ossreviewtoolkit.model.config.RepositoryConfiguration
 import org.ossreviewtoolkit.utils.log
 
 /**
- * A fake [PackageManager] for projects that do not use any of the known package managers.
+ * A fake [PackageManager] for projects that do not use any of the known package managers. It is required as in ORT's
```

Comment:
I believe the comment should also mention that it's not only for projects "which do not use any of the known package managers" but also for projects which do not have a definition file in the analyzer root directory.

Reply:
I've improved the comment in that regard.